modules/rostests/kmtests/ntos_mm/NtCreateSection_drv.c

```diff
@@ -1,8 +1,9 @@
 /*
- * PROJECT:         ReactOS kernel-mode tests
- * LICENSE:         LGPLv2.1+ - See COPYING.LIB in the top level directory
- * PURPOSE:         Test driver for NtCreateSection function
- * PROGRAMMER:      Pierre Schweitzer <pierre@reactos.org>
+ * PROJECT:     ReactOS kernel-mode tests
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * PURPOSE:     Test driver for NtCreateSection function
+ * COPYRIGHT:   Copyright 2016-2018 Pierre Schweitzer <pierre@reactos.org>
+ *              Copyright 2018 Serge Gautherie <reactos-git_serge_171003@gautherie.fr>
  */
```

Comment:
Adding your self in the header for useless changes is a bit over-estimated.
Ftr, having your name here means you can handle bugs that could be triggered in that file. I doubt so.

Reply:
> useless changes

We disagree: I still call my work "improvements and fixes", unless proven otherwise...

> means you can handle bugs that could be triggered in that file

Really? I thought "copyright" means only that I contributed part of the code.
And, as a side-effect, I could be responsible if I did something wrong. Not that I suddenly claim being the only owner and must fix everything.